nuxeo-services/login/nuxeo-platform-login-saml2/src/test/java/org/nuxeo/ecm/platform/auth/saml/SAMLAuthenticatorWithMapperTest.java

```diff
@@ -63,12 +64,10 @@
 import org.w3c.dom.Element;
 
 @RunWith(FeaturesRunner.class)
-@Features({ UserMapperFeature.class, DirectoryFeature.class })
+@Features({ PlatformFeature.class, UserMapperFeature.class })
```

Comment:
Isn't UserManagerFeature enough?

Reply:
Please remove the unused import [here](https://github.com/nuxeo/nuxeo/blob/ed94764987aefe203c11c25be1e05739e8185524/nuxeo-services/login/nuxeo-platform-login-saml2/src/test/java/org/nuxeo/ecm/platform/auth/saml/SAMLAuthenticatorWithMapperTest.java#L42) as know we don't use the `DirectoryFeature`

Reply:
In fact `UserMapperFeature` depends on `PlatformFeature`, so yes `UserManagerFeature` is enough even `UserMapperFeature`. This is a case we talked about in previous comment.